src/hotspot/share/runtime/java.cpp

```diff
@@ -156,6 +157,12 @@ static void print_method_profiling_data() {
   }
 }
 
+void log_vm_stats(outputStream *st) {
+  LogStreamHandle(Info, perf, class, link) log;
+  if (log.is_enabled()) {
+    ClassLoader::print_counters(st);
+  }
```

Comment:
Probably worth adding a comment here as to why we actually print to the passed in stream and not the log stream., given we check if the log stream is enabled. Someone could easily think this is a typo/bug.

Reply:
I can change `log_vm_stats` to accept a `bool` argument so that the the `st` becomes clear.

```
void log_vm_stats(bool use_tty) {
  LogStreamHandle(Info, perf, class, link) log;
  if (log.is_enabled()) {
    outputStream* st = use_tty ? tty : &log;
    ClassLoader::print_counters(st);
  }
}
```

Reply:
That makes the tty usage a lot clearer - thanks.